{
    "checksum": "bc0a5c580b900e2451717d8d54514d2239d23da0970960e01cfd304a5d80e0b7",
    "file": "exceptions.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 1,
    "package": "dnf",
    "path": "dnf/4.5.2-6/dnf/exceptions.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/dnf/4.5.2-6/",
        "metric": {
            "size": 10556
        },
        "pts_link": "https://tracker.debian.org/pkg/dnf",
        "sloc": [
            [
                "python",
                26916
            ],
            [
                "xml",
                778
            ],
            [
                "sh",
                131
            ],
            [
                "makefile",
                42
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/fepitre/dnf",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/d/dnf/4.5.2-6/dnf/exceptions.py",
    "stat": {
        "perms": "rw-r--r--",
        "size": 5849,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "4.5.2-6"
}